docker-production.yml

```diff
@@ -50,9 +50,11 @@ services:
         parallelism: 0
         failure_action: rollback
       update_config:
-        # Update both containers at once, rollback on failure
-        parallelism: 2
+        # Update one containers at once, rollback on failure
```

Comment:
"Stop all existing containers, then update one container at a time, waiting 20s between each. If something fails, rollback to last-known-good"

Reply:
Can you do this same set of changes on `docker-staging.yml` too, so they match?

Reply:
Of course. I should have thought of that.